doc/pyang.1.dbk

```diff
@@ -937,6 +971,108 @@
     </variablelist>
   </refsect1>
 
+  <refsect1 xml:id="man.1.pyang.sid_options">
+    <title>YANG Schema Item iDentifiers (SID)</title>
+    <para>
+      YANG Schema Item iDentifiers (SID) are globally unique unsigned integers
+      used to identify YANG items. SIDs are used instead of names to save space
+      in constrained applications such as COREconf. This plugin is used to
+      automatically generate and updated .sid files used to persist and
+      distribute SID assignments.
+    </para>
+    <para>
+      Options for generating, updating and checking .sid files:
+    </para>
+    <variablelist>
+      <varlistentry>
+        <term><option>--generate-sid-file</option></term>
```

Comment:
you should document the arguments to this option.  I tried "pyang --generate-sid-file foo,yang" and it just hangs.

Reply:
pyang hangs on the following stdin.read().
If you quickly understand why, please send my a propose fix.

    if o.hello:
        ctx.capabilities = hel.registered_capabilities()
        for (mn,rev) in hel.yang_modules():
            mod = ctx.search_module(0, mn, rev)
            if mod is None:
                emarg = mn
                if rev: emarg += "@" + rev
                sys.stderr.write(
                    "module '%s' specified in hello not found.\n" % emarg)
                sys.exit(1)
            modules.append(mod)
    else:
        if len(filenames) == 0:
            text = sys.stdin.read()

Reply:
My point is that the documentation seems to indicate that it is ok to pass just "--generate-sid-file" <filename>, but the code will treat <filename> as an argument to "--generate-sid-file", and then wait for an input module from stdin.

So the behavior is as expected, but the documentation is not.